### PR TITLE
Add Fedora client tag setup to build root script.

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by jlsherrill** [comment]
ACK

File: rel-eng/koji_create_buildroots.sh [hunk 1]
```diff
@@ -86,6 +86,46 @@ koji-katello add-group-pkg katello-$VERSION-rhel5-build srpm-build $SRPM_BUILD_P
...
+#Fedora 20 Client
+
+koji-katello clone-tag katello-nightly-fedora20 katello-$VERSION-fedora20
+
+koji-katello add-tag katello-$VERSION-fedora20-build  --arches=x86_64,i386
+koji-katello add-tag-inheritance katello-$VERSION-fedora20-build katello-$VERSION-fedora20
+koji-katello add-target katello-$VERSION-fedora20 katello-$VERSION-fedora20-build
+
+koji-katello add-external-repo fedora21 -t katello-$VERSION-fedora20-build
+koji-katello add-external-repo fedora21-updates -t katello-$VERSION-fedora20-build
```
Comment (stbenjam) [inline rel-eng/koji_create_buildroots.sh]: This should be fedora 20, right?

**Comment by stbenjam** [comment]
And I guess before I run this for 2.3, fedora20 and 21 need to exist on Koji?

**Comment by ehelms** [comment]
Good catch, I updated it.

**Comment by ehelms** [comment]
We already do F20 and F21 in nightly. So you should be fine to do this (the mash scripts are already cloned).

**Comment by stbenjam** [comment]
Ok cool, tested and works so ACK from me

**Pull request merged**